Languages/Android/questions.json

```diff
@@ -550,6 +550,33 @@
       "trueOrFalse": false,
       "shortAns": "It attaches and detaches fragments from their host activity.",
       "tag": "fragment"
+    },
+    {
+      "id": 61,
+      "question": "To pass an object to different activities, which interface should the object implement?",
+      "details": "To pass an object to different activities, the object should implement either Serializable or Parcelable interface.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "It should implement either serializable or parcelable.",
```

Comment:
This question and shortAns are way too obvious. Please phrase the shortAns as "It is ..." and please dont use the wording of the question itself in the answer, that is a dead give away.